modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,32 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='how-to-use-the-assisted-installer_{context}']
+= How to use the Assisted Installer
+
+{product-title} {ai-full} is a user-friendly installation solution offered on link:http://console.redhat.com[console.redhat.com]. {ai-full} supports the various deployment platforms with a focus on bare metal. It comprises two key components:
+
+* *Assisted Service:* The Assisted Service provides for the hosting of installation artifacts, including:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The web user interface
+  - The REST API
+
+* *{ai-full}*: The {ai-full} provides the installation functionality, and it has following advantages:
+  - *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration file manually.
+  - *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a random node within the cluster.
+  - *Simplified deployment:* Deployment does not require in-depth knowledge of {product-title}.
```

Comment:
I would add also "rich pre flight validation and reasonable defaults"

Reply:
what about mentioning a few of the main features:
SNO
vsphere
dual stack
static ip

Reply:
I've added those comments. Let me know what you think.